presto-orc/src/main/java/com/facebook/presto/orc/OrcOutputBuffer.java

```diff
@@ -386,12 +388,14 @@ public String toString()
 
     private void ensureWritableBytes(int minWritableBytes)
     {
+        assert slice.length() <= maxBufferSize;
```

Comment:
In my opinion this assert should be minWritableBytes <= maxBufferSize . sliceLength <= maxBufferSize will be verified in writeChunkToOutputStream.